src/Microsoft.DotNet.Wpf/src/PresentationUI/MS/Internal/Documents/Application/DocumentStream.cs

```diff
@@ -803,8 +803,10 @@ protected override void Dispose(bool disposing)
 #if DEBUG
             Invariant.Assert(
                 ((i != 3) || (temporary != null) || (inSameFolder)),
-                "Unable to create a temp file.\n"
-                + "Unless IE Cache is read-only we have a defect.");
+                """
+                Unable to create a temp file.
```

Comment:
Will it become `\r\n` ?

Reply:
Yes, but it's an exception message, so I don't think the change is a problem